yarn.lock

```diff
@@ -2598,6 +2598,15 @@
     "@polkadot/api" "^1.0.0-beta.15"
     "@polkadot/types" "^1.0.0-beta.15"
 
+"@polkadot/api-derive@^1.0.0-beta.17":
```

Comment:
Not sure why, but now there are 2 version of `@polkadot/api` in yarn.lock. By experience, this often causes hidden surprises